README.md

```diff
@@ -100,7 +100,7 @@ hal config version edit --version branch:master
 
 # Connect your AWS and ECS accounts
 hal config provider aws account add my-aws-devel-acct \
-    --account-id 123456789012 \
+    --account-id {your-aws-account-id} \
```

Comment:
I like keeping the fake account ID because then you can search and replace in the whole directory locally to customize it: the sample demo and sample pipelines also have the fake account ID.  I'd say at least replace with an environment variable $AWS_ACCOUNT_ID, but that doesn't work for the sample pipelines

Maybe it would be better to have an instruction around doing `sed -i "s/123456789012/your-aws-account-id/g" *` to begin with?  Same for region - eu-central-1 is scattered all over the place.

Reply:
Made a comment to change the account ids as well as repository addresses.